### ISSUE TITLE
Can we train on Companies PDF documents

### ISSUE THREAD

**Issue by skprasadu** [allenai/vila#35]
Hello Shannon,  

I an consulting with companies and they have PDF corpus, and currently we are using Cloud tools that extract these PDF, the results are ok, but very expensive. 

Do you think we can collaborate on this and build Layout aware PDF, your tool seems to be promising, can we train on these PDFs. 

Let me know what you think. I am really interested in collaborating on this.

Krishna

**Comment by lolipopshock** [comment]
Yes, it should be straightforward to do so, as long as you have sufficient labeled data.
